### PR TITLE
doxygen: added community processes (import from Wiki)

### PR DESCRIPTION
**Pull request by emmanuelsearch**
### Contribution description

Everything is in the title ;)

### Testing procedure

Docs should build fine!

### CODE REVIEW

File: doc/doxygen/src/community-processes.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+*Task Forces* are (typically short-lived) efforts to achieve a well-defined goal.
+This goal might be either designing a concept and/or implementing such a concept
+or improving the quality of a certain feature/property of RIOT.
+
+Each Task Force consists of a small number of RIOT developers and has its own Wiki
+page. One or two shepherds are identified per task force as the main contact(s)
+for this activity.
+
+**Creating a new Task Force**. If you wish to launch a new task force, go ahead,
+create your wikipage, name your shepherd(s), and signal it on the forum. RIOT
```
Comment (dylad) [inline doc/doxygen/src/community-processes.md:76]: Question:
Do we want to keep the wiki for this ? Do we have alternative ?

File: doc/doxygen/src/community-processes.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+in the end ;).
+
+Coordinators can propose to give coordinator status to contributors that have
+been noticed as particularly active in this domain. The decision to grant this
+status is then taken via consensus among coordinators. If there is consensus
+on granting the status to a particular contributor, a coordinator will contact
+personally this contributor to propose the status, which the contributor can
+then accept (or turn down).
+
+To contact coordinators, the best is to email the genericmailing list
```
Comment (dylad) [inline doc/doxygen/src/community-processes.md:59]:
```suggestion
To contact coordinators, the best is to email the generic mailing list
```

File: doc/doxygen/src/community-processes.md (hunk 1 above)
Comment (maribu) [reply doc/doxygen/src/community-processes.md:76]: I agree that we should reconsider this. Also the reference to the mailing list.

IMO we should however rather get this in without changes other than typo fixing and do any updates as follow-up PRs. That would IMO make the git history cleaner, if this is basically just an import.

**Review by maribu — APPROVED** [review]
Please squash the typo fix right away.

**Pull request merged**